lib/brcobranca.rb

```diff
@@ -96,14 +96,16 @@ def self.setup
 
   # Módulo para classes de boletos
   module Boleto
-    autoload :Base,          'brcobranca/boleto/base'
+    autoload :Base, 'brcobranca/boleto/base'
+    autoload :BaseSemOverride, 'brcobranca/boleto/base_sem_override'
     autoload :BancoBrasil,   'brcobranca/boleto/banco_brasil'
     autoload :Itau,          'brcobranca/boleto/itau'
     autoload :Hsbc,          'brcobranca/boleto/hsbc'
     autoload :Bradesco,      'brcobranca/boleto/bradesco'
     autoload :Caixa,         'brcobranca/boleto/caixa'
     autoload :Sicoob,        'brcobranca/boleto/sicoob'
     autoload :Sicredi,       'brcobranca/boleto/sicredi'
+    autoload :SantanderComRepasseDeRegras, 'brcobranca/boleto/santander_com_repasse_de_regras'
```

Comment:
O que acha de colocar `SantaderWebservice` ?

Reply:
Então :blush:, O **Brcobranca** deve ser agnóstico, a responsabilidade dele é receber os dados e gerar boletos eu posso repassar esses dados através de um arquivo txt, banco de dados ou etc.. então não faz sentido definirmos e limitarmos como *Webservice*.

Reply:
Então vc teria alguma outra ideia de nome?
Esse nome pra mim não faz muito sentido :thinking:

Reply:
Concordo com o Laerte nessa. Estamos assumindo que o boleto pode vir de várias fontes, mas não sabemos. 
Neste caso, estamos especificamente lidando com um boleto webservice. Se houver o caso de receber de outra forma, acho válido deixar genérico.

Reply:
Conversamos, @laerte-guimaraes, @augustohub  e chegamos a conclusão que o melhor nome seria `SantanaderV2`.

Reply:
@thiagopecanha Faria sentido ter um nome parecido com `SantanderHandler`?

Reply:
Renomeamos para SantanderV2, it's done.